package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.2.45",
+  "version": "1.3.0",
```

Comment:
New year, figured why not bump to 1.3? We should really define a release process around when we increment based on some sort of major/minor/revision logic. 

For now v1.3 is just marking the start of 2023 our potential rebranded color palette!

Reply:
i agree that a new color pallete is an excellent reason to push a "minor" upgrade!

2.0.0 should mean "some things from 1.x.x are no longer available." An example might be any of the X2 migrations being finalized, so Type2 is no longer available after the upgrade to `2.0.0`